protocols/gossipsub/src/handler.rs

```diff
@@ -504,7 +467,8 @@ impl ConnectionHandler for Handler {
                                 Some(OutboundSubstreamState::WaitingOutput(substream))
                         }
                         Poll::Ready(Err(e)) => {
-                            return Poll::Ready(ConnectionHandlerEvent::Close(e))
+                            log::info!("NegotaitedSubstream error: {e}");
```

Comment:
Can we make these according to the function that failed? In this case "flushing". Like "failed to flush outbound stream".